src/surrogate_heat_driver.cpp

```diff
@@ -114,13 +315,22 @@ void SurrogateHeatDriver::solve_step()
 
   for (int i = 0; i < n_pins_; ++i) {
     for (int j = 0; j < n_axial_; ++j) {
+      // approximate cladding surface temperature as equal to the fluid
+      // temperature, i.e. this neglects any heat transfer resistance
+      double T_co = fluid_temperature_(i, j);
+
+      // Set initial temperature to surface temperature
+      for (auto& T : temperature_) {
+        T = T_co;
+      }
```

Comment:
I think xtensors have a fill method so you could do:
```C++
temperature_.fill(T_co);
```

Reply:
nice, didn't know about that